src/components/notebook/chem-test.tsx

```diff
@@ -106,11 +106,13 @@ export const ChemTest: React.FC<IProps> = (props) => {
                                                 ((index === stepsComplete) &&
                                                   (index === testResult.current?.currentStep)),
                                       finished: index < stepsComplete,
-                                      running: index === testResult.current?.currentStep && index === stepsComplete})}
+                                      running: index === testResult.current?.currentStep && index === stepsComplete,
+                                      smallfont: chemistryTest.type === ChemTestType.airTemperature ||
+                                                chemistryTest.type === ChemTestType.waterTemperature})}
```

Comment:
This really should be part of a button group component that adjusts its font size based on its text. Which tests require the smaller font size will be language-dependent. But this is fine in the interim.

Reply:
Yeah, there's a bunch of places where text size checks would be needed if we ever really wanted to use the translation.